diff_withfortios

```diff
@@ -0,0 +1,47 @@
+diff --git a/TEMP_Cert_1.pem b/TEMP_Cert_1.pem
```

Comment:
is there a way to not have this .pem in the repo?
Is this diff file even supposed to be checked in?
You might have to rebase and force push to get rid of it

Reply:
Hi Jamie, i am now using the certificate imported by csfd for the remote fortigate in my SAML configuration. No need to use certificate for this case anymore.